projects/ngx-favicon/src/lib/ngx-favicon.interface.ts

```diff
@@ -0,0 +1,5 @@
+export interface NgxFaviconConfig<CustomFavicon> {
+  targetId: string;
+  default: string;
+  custom: CustomFavicon;
```

Comment:
To be clear to the developer, make the default field have `Url` suffix

Reply:
Done